Fix build with Meson 0.50.0 and above.

Taken from upstream repository:
https://gitlab.gnome.org/GNOME/at-spi2-core/commit/44a812ea51223d82f21a098a2d45fcc5c329ce7a

diff --git a/atspi/meson.build b/atspi/meson.build
index b7a9357df1c4857dffa2ec123cd3073e67a07fea..2a6915d76202dabf923b50e6a8b85cbb83cb43d0 100644
--- a/atspi/meson.build
+++ b/atspi/meson.build
@@ -57,7 +57,7 @@ atspi_headers = [
 
 atspi_includedir = join_paths(get_option('prefix'), get_option('includedir'), 'at-spi-2.0', 'atspi')
 
-install_headers(atspi_headers, subdir: atspi_includedir)
+install_headers(atspi_headers, install_dir: atspi_includedir)
 
 atspi_enums = gnome.mkenums('atspi-enum-types',
                             sources: [ 'atspi-constants.h', 'atspi-types.h' ],
